src/client/client4.js

```diff
@@ -564,11 +564,18 @@ export default class Client4 {
         );
     };
 
-    getProfilesNotInTeam = async (teamId, page = 0, perPage = PER_PAGE_DEFAULT) => {
-        this.trackEvent('api', 'api_profiles_get_not_in_team', {team_id: teamId});
+    getProfilesNotInTeam = async (teamId, groupConstrained, page = 0, perPage = PER_PAGE_DEFAULT) => {
+        this.trackEvent('api', 'api_profiles_get_not_in_team', {team_id: teamId, group_constrained: groupConstrained});
+
+        let queryStringObj;
+        if (groupConstrained) {
+            queryStringObj = {not_in_team: teamId, page, per_page: perPage, group_constrained: true};
+        } else {
+            queryStringObj = {not_in_team: teamId, page, per_page: perPage};
+        }
```

Comment:
Don't necessarily need to duplicate the whole object definition in both conditional branches when only `group_constrained` differs.